src/macaron/slsa_analyzer/analyzer.py

```diff
@@ -1047,6 +977,34 @@ def perform_checks(self, analyze_ctx: AnalyzeContext) -> dict[str, CheckResult]:
 
         return results
 
+    def _resolve_more_dependencies(self, dependencies: dict[str, DependencyInfo]) -> None:
```

Comment:
This function and `resolve_dependencies` can be moved to the `dependency_resolver` module to keep the dependency resolution code in one place.